Marlin/Marlin_main.cpp

```diff
@@ -3279,6 +3376,21 @@ inline void gcode_G28() {
 
           measured_z = probe_pt(xProbe, yProbe, z_before, act, verbose_level);
 
+          #if ENABLED(REPROBE)
+            if(measured_z == MIN_PROBE_PT) {
```

Comment:
A space is preferred between `if` and opening parenthesis. Also between `for` and opening parenthesis. See https://github.com/MarlinFirmware/Marlin/wiki/DNE-Coding-Standards